Python/Product/Profiling/PythonProfilingPackage.cs

```diff
@@ -44,7 +44,7 @@ namespace Microsoft.PythonTools.Profiling {
     // This attribute tells the PkgDef creation utility (CreatePkgDef.exe) that this class is
     // a package.
     [PackageRegistration(UseManagedResourcesOnly = true)]
-    [Description("Python Tools Profiling Package")]
+    [Description("Python Tools Profiling Package")] // TODO: Localization: does this do anything?
```

Comment:
No idea. Let's just leave it unlocalized for now - it'll either have no impact or we'll get a loc bug on it.